content-src/components/DiscoveryStreamComponents/SafeAnchor/SafeAnchor.jsx

```diff
@@ -19,9 +19,9 @@ export class SafeAnchor extends React.PureComponent {
   }
 
   render() {
-    const {url, className, onLinkClick} = this.props;
+    const {url, className, key, onLinkClick} = this.props;
```

Comment:
Pretty sure `key` doesn't even make it to this component. We do want to set it on the component, but React handles the special prop and doesn't pass it in.

Reply:
good to know, will remove redundant key assignment from SafeAnchor.

Reply:
@Mardak PR updated with feedback, waiting for test to go green before merging. Thanks!